src/services/export.js

```diff
@@ -146,7 +144,25 @@ const replaceElementsWithScreenshots = async (elements, page) => {
   // eslint-disable-next-line no-restricted-syntax
   for (const element of elements) {
     // eslint-disable-next-line no-await-in-loop
-    await page.evaluate(replaceElementWithScreenshot, element, TMP_PATH);
+    await page.evaluate(replaceElementWithScreenshot, element, TMP_FOLDER);
+  }
+};
+
+/* eslint no-param-reassign: ["error", { "props": true, "ignorePropertyModificationsFor": ["el"] }] */
+const adjustElementHeight = el => {
+  const height = el.clientHeight;
+  el.style.height = `${height}px`;
+};
+
+const adjustElementsHeight = async (elements, page) => {
```

Comment:
Minor comment to rename this to either `adjustElementsHeights` or better `adjustHeightForElements`. It just reads a bit better. `adjustElementsHeight` could be interpreted as adjust element's height, which is the same as `adjustElementHeight`. Just to avoid confusion, better rename it.